db/migrate/20160527093031_limit_os_description.rb

```diff
@@ -0,0 +1,12 @@
+class LimitOsDescription < ActiveRecord::Migration
+  def up
+    Operatingsystem.all.each do |os|
+      os.update_attribute(:description, os.description.truncate(255))
```

Comment:
This fails when the os description is NULL/nil, it probably needs to check for presence:

<pre>
undefined method `truncate' for nil:NilClass
/home/dcleal/code/foreman/foreman/db/migrate/20160527093031_limit_os_description.rb:4:in `block in up'
/home/dcleal/.rvm/gems/ruby-2.3.0@foreman/gems/activerecord-4.2.6/lib/active_record/relation/delegation.rb:46:in `each'
/home/dcleal/.rvm/gems/ruby-2.3.0@foreman/gems/activerecord-4.2.6/lib/active_record/relation/delegation.rb:46:in `each'
/home/dcleal/code/foreman/foreman/db/migrate/20160527093031_limit_os_description.rb:3:in `up'
</pre>